kernel/Slippi.c

```diff
@@ -359,7 +318,7 @@ static u32 SlippiHandlerThread(void *arg) {
 		UINT wrote;
 		f_lseek(&currentFile, f_size(&currentFile));
 		f_write(&currentFile, slippi_msg->ioctl.buffer_io, slippi_msg->ioctl.length_io, &wrote);
-		memcpy((SlipMem + SlipMemCursor), slippi_msg->ioctl.buffer_io, slippi_msg->ioctl.length_io);
+		memcpy(&SlipMem[SlipMemCursor], slippi_msg->ioctl.buffer_io, slippi_msg->ioctl.length_io);
```

Comment:
`&SlipMem` doesn't dereference the pointer. I think `SlipMem[offset]` or `(SlipMem + offset)` would be the correct usage.